src/gsy_e/gsy_e_core/user_profile_handler.py

```diff
@@ -299,7 +308,8 @@ def update_time_and_buffer_profiles(self, timestamp: DateTime, area: "Area") ->
         """ Update current timestamp and get the first chunk of data from the DB"""
         self._update_current_time(timestamp)
         if self.db:
-            uuids_used_in_setup = self._get_profile_uuids_from_setup(area, [])
+            uuids_used_in_setup = self._get_profile_uuids_from_setup(area)
+            print(uuids_used_in_setup)
```

Comment:
![you-forgot-me-joey](https://user-images.githubusercontent.com/31365000/196380815-c4611bdc-e80e-4474-8c3f-6c2f99bf76c2.gif)

Reply:
I think it is time for me to put some money in the leftover-print-piggybank.